addons/account_payment/__init__.py

```diff
@@ -2,4 +2,3 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import controllers
-from . import models
```

Comment:
robodoo r-

Reply:
Hello, complaints must be formatted as memes before submission. KR